pkgs/applications/science/physics/crystfel/default.nix

```
@@ -169,13 +170,29 @@ let
       "-DENABLE_BZIP2_PLUGIN=yes"
     ];
   };
+
+  millepede-ii = stdenv.mkDerivation rec {
+    pname = "millepede-ii";
+    version = "V04-13-06";
```

```suggestion
    version = "04-13-06";
```
we strip version prefixes like v, REL or the package name.

---

Ah, wasn't aware of that. Fixed.